espnet2/asr/preencoder/linear.py

```diff
@@ -19,19 +18,21 @@ def __init__(
         self,
         input_size: int,
         output_size: int,
+        dropout: float = 0.0
     ):
         """Initialize the module."""
         assert check_argument_types()
         super().__init__()
 
         self.output_dim = output_size
         self.linear_out = torch.nn.Linear(input_size, output_size)
+        self.dropout = torch.nn.Dropout(dropout)
```

Comment:
this should be fine

Reply:
I think it makes more sense to use dropout on pre-encoder rather than using spec-augment on wavlm features . Dropout is on input right now.

Reply:
I think it is fine. Did you compare how much difference in the performance could be?

Reply:
It is the same, I have used in the previous experiments and was on par with the IRIS way. But I feel it is more correct